draft-ietf-quic-transport.md

```diff
@@ -198,6 +198,8 @@ strengths of QUIC include:
 
 * Version negotiation
 
+* Support for monitoring
```

Comment:
If you completely remove the question, then the bit description just appears out of nowhere. But then, yes, we could be specific about latency monitoring, rather than monitoring in general.